planning_domains_api.py

```diff
@@ -104,6 +104,33 @@ def simple_query(qs):
     else:
         return res['result']
 
+def create_col(name, description, domain_set, ipc):
+    """Create a new collection in the database"""
+    attribute = ''  # Unknown for now
+    value = ''      # Same Unknown for now
+
+    params = {'user': USER_EMAIL,
+              'password': USER_TOKEN,
+              'ipc': ipc,
+              'desc': description,
+              'domain_set': domain_set
```

Comment:
Worth checking to see if all the domains in the `domain_set` exist before putting this into the DB.